solmanager.config.sample

```diff
@@ -1,24 +1,18 @@
-[solmanager]
-# Connection
-serialport = /dev/ttyUSB3
-tcpport = 8080
-
-# Application
-filecommitdelay = 60
-sendperiodminutes = 1
-fileperiodminutes = 1
-
-# Security
-token = DEFAULT_SOLMANAGERTOKEN
+[application]
+logfile             = solmanager.log
+backupfile          = solmanager.backup
+serialport          = /dev/ttyUSB3
+tcpport             = 8080
+filecommitdelay     = 60
```

Comment:
difference between `filecommitdelay` and `fileperiodminutes`?
Please add comments to every line